### PR TITLE
Fixes for post-625

### PR DESCRIPTION
**Pull request by tfennelly**
@jglick hopefully these fix your issues on that branch.

### CODE REVIEW

File: support/src/main/resources/org/jenkinsci/plugins/workflow/support/steps/build/BuildTriggerStep/config.jelly [hunk 1]
```diff
@@ -40,20 +40,20 @@ THE SOFTWARE.
...
-                onSuccess : function(x) {
-                    div.innerHTML = x.responseText;
-                    Behaviour.applySubtree(div);
-                },
-                onFailure : function(x) {
-                    div.innerHTML = "<b>ERROR</b>: Failed to load parameter definitions: " + x.statusText;
-                }
-            });
-        }
-    </script>
```
Comment (tfennelly) [inline support/src/main/resources/org/jenkinsci/plugins/workflow/support/steps/build/BuildTriggerStep/config.jelly]: So the issue with this ... once this gets assembled, the content returned to the browser has the `<script>` tags in the middle of the config form `<table>` elements (between the `<tr>`s), causing HtmlUnit to balk on it.

File: aggregator/src/test/java/org/jenkinsci/plugins/workflow/steps/input/InputStepTest.java [hunk 2]
```diff
@@ -62,7 +62,8 @@
 public class InputStepTest extends Assert {
     @Rule
     public JenkinsRule j = new JenkinsRule() {
-        // TODO https://github.com/jenkinsci/jenkins/pull/1774/files#diff-a388b7a6a62410e46234342a32e29c02L1315 seems to have broken the super impl
+        // TODO Remove post core 1.639
+        // See https://issues.jenkins-ci.org/browse/JENKINS-31631 and https://github.com/jenkinsci/jenkins/commit/7fdb589133de09e77d29397ee01bcc9004e25cb2
```
Comment (jglick) [inline aggregator/src/test/java/org/jenkinsci/plugins/workflow/steps/input/InputStepTest.java]: Should I guess rather link to https://github.com/jenkinsci/jenkins/pull/1918.

**Comment by jglick** [comment]
Beautiful, :bee: for @reviewbybees

**Comment by amuniz** [comment]
:bee:

**Pull request merged**